pyvista/examples/downloads.py

```diff
@@ -4561,7 +4459,7 @@ def download_black_vase(load=True, progress_bar=False):  # pragma: no cover
     Original datasets are under the CC BY 4.0 license.
 
     For more details, see `Ivan Nikolov Datasets
-    <https://github.com/pyvista/vtk-data/tree/master/Data/ivan-nikolov>`_
+    <https://github.com/pyvista/vtk-data/tree/master/ivan-nikolov>`_
```

Comment:
All of these changes removing `Data` from the url lead to 401 errors. The original links were correct.